src/js/rtc_session.js

```diff
@@ -77,7 +77,7 @@ export class GrabLocalMediaState extends RTCSessionState {
     onEnter() {
         var self = this;
         var startTime = Date.now();
-        if (self._rtcSession._userAudioStream) {
+        if (self._rtcSession._userProvidedStream) {
```

Comment:
can we rename this var? the name only suggests it's a stream, not a boolean flag.